CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# 0.33, April 2018
+
+New features:
+
+* Context pre-initialization with TruffleRuby `--native`, which significantly
+  improves the startup time.
```

Comment:
While not technically wrong, saying "improve startup time" sounds more natural to me than "improves the startup time".